samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
> We can't add arbitrary claims to an identity

Of course we can - that's exactly what claims transformation *is*. Plus, by setting the issuer to an internal value the user can tell this is added by us. You SID example is disingenuous as we're not adding SIDs. Or Roles.